avm/ptn/azd/acr-container-app/main.bicep

```diff
@@ -0,0 +1,253 @@
+metadata name = 'avm/ptn/azd/acr-container-app'
```

Comment:
This will be used as the readme title. I'd suggest a more user friendly short name, possibly starting with `Azd`

Reply:
@eriqua I have updated by your comments.